ReactNativeClient/lib/shim-init-node.js

```diff
@@ -357,6 +357,16 @@ function shimInit() {
 		return Buffer.byteLength(string, 'utf-8');
 	};
 
+	shim.readFileToBlob = async function(path) {
```

Comment:
That should be in fsdriver. But what is the purpose of it? It seems similar to fsDriver.readFile?

Reply:
Glad you brought this up, I wasn't sure if I needed to handle deal with binary files on React Native differently. But `readFile` would be simplier.